<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetFirst</class>
 <widget class="QDialog" name="SetFirst">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>27</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/image/image/setting.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>144</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/image/image/return.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>20</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="movieButton">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>8</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>开机录像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="audioButton">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>8</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>录音</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ADASButton">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>92</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>ADAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reverseButton">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>92</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>倒车线</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>录像时间设置</string>
   </property>
   <widget class="QSpinBox" name="movieTimeSetting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>分辨率设置</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1920×1080</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280×720</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QSlider" name="lightSlider">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>50</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="voiceSlider">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>50</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lightLabel">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>27</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="voiceLabel">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>27</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lightButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>167</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="voiceButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>167</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
